lib/kubernetes-deploy/bindings_parser.rb

```diff
@@ -7,7 +7,15 @@ module KubernetesDeploy
   module BindingsParser
     extend self
 
-    def parse(string)
+    def parse(binds)
```

Comment:
I think this PR is 👍 functionality-wise, but I feel like this class's API is a bit weird now--it's not intuitive what the difference is between `BindingsParser::parse` and `BindingsParser::parse_binding` would be.

Here's a suggestion:

```ruby
# If you need multiple, you do:
bindings = KubernetesDeploy::BindingsParser.new
# ...
  opts.on("--bindings=BINDINGS", "...") { |b| bindings.add(b) }
# ...
bindings.parse

# But you can still also use the exact same API as before:
KubernetesDeploy::BindingsParser.parse(bindings_string)

module KubernetesDeploy
  class BindingsParser
    def self.parse(string)
      new(string).parse
    end

    def initialize(initial_string=nil)
      @raw_bindings = Array(initial_string)
    end

    def add(string)
      @raw_bindings << string
    end

    def parse
      result = {}
      @raw_bindings.each do |string|
        # parse/merge
      end
      # etc
      result
    end
```

Reply:
Makes sense, refactored